src/main/java/teammates/ui/controller/InstructorCourseInstructorEditSaveAction.java

```diff
@@ -55,6 +77,27 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         return result;
     }
 
+    /**
+     * This method gets the numbers of instructors that would be made visible to students if
```

Comment:
1. `This method gets` -> `Returns` 
2. Wouldn't it be simpler to count the number of _currently_ visible instructors? Otherwise the method abstractions is a bit odd and the method name is misleading.